scheduler/src/main/java/io/camunda/zeebe/scheduler/ActorJob.java

```diff
@@ -59,10 +62,25 @@ void execute(final ActorThread runner) {
         schedulingState = TaskSchedulingState.TERMINATED;
       } else {
         schedulingState = TaskSchedulingState.QUEUED;
+        scheduledAt = System.nanoTime();
       }
     }
   }
 
+  private void observeSchedulingLatency(final ActorMetrics metrics) {
```

Comment:
:wrench: I somehow expected the enabled flag here tbh

Reply:
If we have it disabled it is just unnecessary work we do here.

Reply:
Good point! Then I can also add a check in line 65 and save an unnecessary call to `nanoTime`